Wordle.Api/Wordle.Api/Models/AppUser.cs

```diff
@@ -9,5 +9,7 @@ public class AppUser : IdentityUser
         public double AverageAttempts { get; set; }
 
         public int AverageSecondsPerGame { get; set; }
+
```

Comment:
Add MasterOfTheUniverse here, and you're super close to getting full credit!

Reply:
Or possibly have a claim object, since that includes the age and MasterOfTheUniverse

Reply:
I don't think I technically need that here? - Claims are handled by another file, such calculating the AppUser's age from this birthday variable (the only reason it's here). I might have totally missed something, but as far as I can tell everything works as indented as-is.